CONTRIBUTING.md

```diff
@@ -185,6 +185,18 @@ pytest test/test_nn.py -k Loss -v
 The above is an example of testing a change to Loss functions: this command runs tests such as
 `TestNN.test_BCELoss` and `TestNN.test_MSELoss` and can be useful to save keystrokes.
 
+### Reproducing Linux CI errors locally
+1. If CUDA is needed, run `export USE_CUDA_DOCKER_RUNTIME=1`.
+2. Run `source .circleci/scripts/setup_ci_dependencies.sh` to set up local environment.
+3. In the CI build log, find the line that starts with "Docker image download link: ",
+use the link to download the Docker image tar file `<DOCKER_IMAGE>.tar`.
```

Comment:
Can we make the CI jobs print these instructions on failure? It can be something similar to how we make sure sccache stats get printed at the end of jobs, even if they fail. Most users aren't going to find the instructions here; if we put them on the CI logs themselves, that makes them much more discoverable.